linuxNew/jdk/debian/src/packageTest/java/packaging/DebianFlavours.java

```diff
@@ -38,7 +38,7 @@
 			Arguments.of("debian", "bullseye"),
 			Arguments.of("ubuntu", "bionic"),
 			Arguments.of("ubuntu", "focal"),
-			Arguments.of("ubuntu", "hirsute")
```

Comment:
You likely didn't intend to remove `hirsute` here?

Reply:
so you prefer to keep it?

Reply:
Yes please